0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -155,7 +154,8 @@ The mapping from the ILP address to the local ledger address is dependent on the
 #### getBalance
 <code>ledgerPlugin.getBalance() ⇒ Promise.&lt;String></code>
 
-Return an integer string representing the current balance. Plugin must be connected, otherwise the promise should reject.
+Return an integer string representing the current balance, in the ledger's base unit. For example, if the ledger's `currencyCode` is 'EUR' and its `currencyScale` is 5, then
```

Comment:
yes. I think in the switch to integers, it changed from a `String` that allows `parseFloat`, to a `String` that allows `parseInt`. At the same time, its absolute value was multiplied by `Math.pow(10, scale)`. This docs change in this PR only adds some extra clarifying text about the new situation, it doesn't change the behavior.